pages/pools/index.tsx

```diff
@@ -127,9 +136,13 @@ const Pools: FC = () => {
 			);
 		}
 
+		if (process.env.NODE_ENV === 'production') {
+			list = list.filter(({ network }) => network === 'mainnet' || network === 'optimism');
```

Comment:
we should prob create an enum for the network names that we are using for the urls and use them here and in the object we are exporting for that in case we change the names in the future